src/services/FulfillmentService.js

```diff
@@ -47,10 +49,18 @@ class FulfillmentService {
         params: { slug },
         body: { type, value },
       } = req;
-
+      const url = `${DOMAINS[node.env.NODE_ENV]}/rest/v1/fulfillments/referrals/${slug.zipCode}`;
```

Comment:
This method is supposed to get referrals by slug, as the name of the function implies. The slug will tell us what "category" of referral the user is asking for. I'm not sure why Eugene called the other values type and slug, but they should just be a location. For now, we should assume that location is the name of the county that the end user is in. So, the request should be for a particular type of referral in a particular county. This is complicated by the fat that there are two different categories of referrals. we have VCSO and MVPN. Let's chat about this.